[Feature] pg backup integrity (#334)
NBISweden/sda-services-backup: README.md

### Backing up a database

#### Dump

* backup will be stored in S3 in the format of `YYYYMMDDhhmmss-DBNAME.sqldump`

```cmd
./backup-svc --action pg_dump
```

#### Pg_basebackup

* backup will be stored in S3 in the format of `YYYYMMDDhhmmss-DBNAME.tar`

```cmd
docker container run --rm -i --name pg-backup --network=host $(docker build -f dev_tools/Dockerfile-backup -q -t backup .) /bin/sda-backup --action pg_basebackup
```

**NOTE**

This type of backup runs through a docker container because of some compatibility issues
that might appear between the PostgreSQL 13 running in the `db` container and the local one.

### Restoring up a database

#### Restore dump file

* The target database must exist when restoring the data.

```cmd
./backup-svc --action pg_restore --name PG-DUMP-FILE
```

#### Restore from physical copy

This is done in more stages.

* The target database must be stopped before restoring it.

* Create a docker volume for the physical copy.

> * Get the physical copy form the S3 and unpack it in the docker volume which has been created in the previous step

Reviewer: ```suggestion
* Get the physical copy from the S3 and unpack it in the docker volume which was created in the previous step
```
